### PR TITLE
balboa-common cleanup

### PR DESCRIPTION
**Pull request by coyotebush**

### CODE REVIEW

File: balboa-common/src/main/java/com/socrata/balboa/metrics/data/QueryOptimizer.java [hunk 1]
```diff
@@ -22,6 +22,8 @@ Period lessGranular(Period current) {
         return current;
     }
 
+    // Determine the optimal way to cover the range [start, end] using available Period types.
```
Comment (mhotan) [inline balboa-common/src/main/java/com/socrata/balboa/metrics/data/QueryOptimizer.java]: Nit: can we make this standard java doc format?

**Comment by jacobsi** [comment]
please retest this

**Comment by jacobsi** [comment]
retest this please

**Comment by jacobsi** [comment]
lgmt

**Pull request merged**